### PR TITLE
feat(upload): Re-work the file/multiple file upload ui

### PR DESCRIPTION
**Pull request by genie-magic**
**What this PR does** 📖
Enhance multiple uploads based on Nick's AP-93 branch
- Make Individual components for Image Upload Preview Item
- Refactor styles & enhance UI
- Fixed broken error message UI

**Which issue(s) this PR fixes** 🔨

AP-265

**Special notes for reviewers** 🗒️
Please approve changes on locales branch before testing this branch. (https://github.com/Satellite-im/locales/pull/17)

### CODE REVIEW

**Comment by nickjjang** [comment]
image
when I upload more than 8 images error message shows like above. will @genie-magic fix number of image in translation message to be 8(not 4)?

**Comment by genie-magic** [comment]
> **Special notes for reviewers** 🗒️
> Please approve changes on locales branch before testing this branch. ([Satellite-im/locales#17](https://github.com/Satellite-im/locales/pull/17))

@nickjjang  Please read 'Special notes for reviewers 🗒️' I wrote.

**Review by nickjjang — APPROVED** [review]

File: components/views/files/upload/Upload.html [hunk 1]
```diff
@@ -8,75 +8,18 @@
...
-                </div>
-              </div>
-            </div>
-          </div>
+          <FilesUploadPreviewItem
+            :item="item"
+            :index="index"
+            :key="index"
+            @removeUploadItem="removeUploadItem(index)"
+            v-for="(item, index) in files"
```
Comment (stavares843) [inline components/views/files/upload/Upload.html:21]: we should add these in locales

File: components/views/files/upload/previewItem/PreviewItem.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <div v-for="itemProgress in currentProgress">
+        <div v-if="itemProgress.name === item.file.name">
+          <div v-if="itemProgress.progress !== 100">
+            <UiProgress
+              :progress="itemProgress.progress"
+              type="loading"
+              class="loading-bar"
+            />
+            <TypographySubtitle
+              :text="`${itemProgress.progress.toFixed(2)}% Uploaded...`"
```
Comment (stavares843) [inline components/views/files/upload/previewItem/PreviewItem.html:43]: we should add these in locales

Comment (genie-magic) [reply components/views/files/upload/previewItem/PreviewItem.html:43]: Okay

**Comment by stavares843** [comment]
Captura de ecrã 2021-12-29, às 16 57 27

JIRA ticket mentions the above

is that to be implemented separately? @genie-magic

**Comment by stavares843** [comment]
added a separate ticket to implement that - [AP-381](https://satellite-im.atlassian.net/browse/AP-381)

**Review by stavares843 — APPROVED** [review]

**Comment by genie-magic** [comment]
@stavares843  Can you explain why you remove this branch?

**Comment by stavares843** [comment]
hello! 👋  @genie-magic 

the PR is merged

Captura de ecrã 2021-12-29, às 17 33 43

when a PR gets merged, the branch is automatically deleted.

Captura de ecrã 2021-12-29, às 17 34 08


the branch can be restored if you need, but all the content from this PR is now on dev branch

**Pull request merged**